src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+
+  const personPartnerInfo = sex === 'm'
+    ? `${partnerName} is my wife`
+    : `${partnerName} is my husband`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {
+        Boolean(age) && (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        )
+      }
+
+      {
+        isMarried ? (
+          <p className="Person__partner">
```

Comment:
Remember about DRY. You use this tag twice for both variants. Why not change only containing this tag depending on `isMarried`